### PR TITLE
KG - Bootstrap EN.YMLization

### PR DESCRIPTION
**Pull request by kyle-glick**

### CODE REVIEW

File: app/views/dashboard/arms/create.js.coffee [hunk 1]
```diff
@@ -23,7 +23,7 @@ $("#modal_errors").html("<%= escape_javascript(render(partial: 'shared/modal_err
 $("#per_patient_services").html("<%= escape_javascript(render(:partial =>'dashboard/sub_service_requests/per_patient_per_visit', locals: {sub_service_request: @sub_service_request, service_request: @service_request})) %>");
 
 $("#sub_service_request_header").html("<%= escape_javascript(render(partial: 'dashboard/sub_service_requests/header', locals: { sub_service_request: @sub_service_request })) %>");
-$("#subsidy_information").html("<%= escape_javascript(render(partial: 'dashboard/subsidies/subsidy', locals: { sub_service_request: @sub_service_request })) %>");
+$("#subsidy_information").html("<%= escape_javascript(render(partial: 'dashboard/subsidies/subsidy', locals: { sub_service_request: @sub_service_request, admin: true })) %>");
```
Comment (amcates) [inline app/views/dashboard/arms/create.js.coffee]: This doesn't look like YMLization.  Should this get removed and put in a proper pull request?

File: app/views/dashboard/line_items/create.js.coffee [hunk 2]
```diff
@@ -23,7 +23,7 @@ $("#modal_errors").html("<%= escape_javascript(render(partial: 'shared/modal_err
 $("#study-level-activities-table").bootstrapTable 'refresh', {silent: true}
 
 $("#sub_service_request_header").html("<%= escape_javascript(render(partial: 'dashboard/sub_service_requests/header', locals: { sub_service_request: @sub_service_request })) %>");
-$("#subsidy_information").html("<%= escape_javascript(render(partial: 'dashboard/subsidies/subsidy', locals: { sub_service_request: @sub_service_request })) %>");
+$("#subsidy_information").html("<%= escape_javascript(render(partial: 'dashboard/subsidies/subsidy', locals: { sub_service_request: @sub_service_request, admin: true})) %>");
```
Comment (amcates) [inline app/views/dashboard/line_items/create.js.coffee]: Same type of question as before

File: app/views/dashboard/arms/create.js.coffee (hunk 1 above)
Comment (kyle-glick) [inline app/views/dashboard/arms/create.js.coffee]: @ctt3 What do you think about this? I think we had put it in so that I could submit the forms.

File: config/locales/en.yml [hunk 3]
```diff
@@ -36,6 +36,505 @@ en:
...
+        text2: "is less than the unit minimum of"
+        text3: "Quantity is being reset to the previous value."
+        text4: "Units Per Quantity of"
+        text5: "is more than the maximum allowed of"
+        text6: "Units Per Quantity is being set to the maximum allowed value."
+      merged:
+        total_study: "Study Cost Totals"
+
+
+
```
Comment (ctt3) [inline config/locales/en.yml]: I like the organization!

File: app/views/dashboard/arms/create.js.coffee (hunk 1 above)
Comment (ctt3) [inline app/views/dashboard/arms/create.js.coffee]: While I agree with Andrew that this fix is out of scope for this particular pull request, it is a necessary change that I'm sure without he was not able to access all the necessary parts of the app to accomplish this task. Either way, this fix needs to come into the bootstrap branch.
